butler.py

```diff
@@ -98,6 +98,118 @@ def _setup_args_for_remote(parser):
   subparsers.add_parser('reboot', help='Reboot with `sudo reboot`.')
 
 
+def _add_weights_fuzzer_subparser(weights_subparsers):
+  """Adds a parser for the `weights fuzzer` command."""
+  parser = weights_subparsers.add_parser(
+      'fuzzer', help='Interact with FuzzerJob weights.')
+  subparsers = parser.add_subparsers(dest='fuzzer_command')
+
+  subparsers.add_parser(
+      'platforms', help='List distinct platform field values.')
+
+  list_parser = subparsers.add_parser('list', help='List FuzzerJob entries.')
+  list_parser.add_argument(
+      '-p',
+      '--platforms',
+      help='Which platforms to list entries for.',
+      nargs='+')
+  list_parser.add_argument(
+      '-f', '--fuzzers', help='Which fuzzers to list entries for.', nargs='+')
+  list_parser.add_argument(
+      '-j', '--jobs', help='Which jobs to list entries for.', nargs='+')
+  list_parser.add_argument(
+      '--format',
+      help='Output format.',
+      choices=['text', 'csv'],
+      default='text')
+
+  aggregate_parser = subparsers.add_parser(
+      'aggregate', help='Aggregate matching FuzzerJob entries.')
+  aggregate_parser.add_argument(
+      '-p', '--platform', help='Which platform to query.', required=True)
+  aggregate_parser.add_argument(
+      '-f', '--fuzzers', help='Which fuzzers to aggregate.', nargs='+')
+  aggregate_parser.add_argument(
+      '-j', '--jobs', help='Which jobs to aggregate.', nargs='+')
+
+
+def _add_weights_batches_subparser(weights_subparsers):
+  """Adds a parser for the `weights fuzzer-batch` command."""
+  parser = weights_subparsers.add_parser(
+      'fuzzer-batch',
+      help='Interact with FuzzerJobs (FuzzerJob batches) weights.')
```

Comment:
Is there a way to describe this better? I'm not sure I understand.

Reply:
How's this?